doc/tutorials/modeling-complex-events-ctim.md

```diff
@@ -0,0 +1,77 @@
+## Modeling Complex Events in CTIM
+
+CTIM is capable of modeling arbitrarily complex event data. That is, events that are triggered by other events, which themselves might be triggered by other events.
+
+## Synopsis
+
+- Arbitrarily complex cybersecurity events can be modeled in CTIM.
+- Pattern matching or detection rules are modeled with the `indicator` entity.
+- An `indicator` entity can be `based-on` other `indicator` entities.
+- When modeling `indicator` entities, less is more.
+  - Don't include the patterns you match against in your `indicator` entities. They'll be used for detection evasion.
+  - The `indicator` entity should only summarize the threat. Using other entities for describing threats in depth.
+- When a pattern matching or detection rule described by an `indicator` is triggered, we create a `sighting` that is a `sighting-of` relationship to that `indicator`.
+- A `sighting` entity can be `based-on` other `sighting` entities, in addition to being a `sighting-of` an `indicator`.
+
+## Indicators as Detection or Pattern Matching Rules
+
+There are two basic indicator types. This tutorial concerns itself with the first type, which is a detection or pattern matching rule. These rules look for patterns in data, and can run inside of rule engines, expert systems, firewalls, log monitors, or any other kind of scanning or monitoring system.
+
+Pattern matching rules are often modeled using composition, such that a series of pattern matches within a given timeframe can combine to trigger more complex pattern matches. Here are three contrived examples:
```

Comment:
Perhaps there's a better word than `composition` here. Aggregation? I had trouble relating the two halves of the sentence.

Reply:
Too many verbs. Tightened the prose.